api/v1/model/src/commonMain/kotlin/OrtRun.kt

```diff
@@ -175,6 +175,13 @@ data class CreateOrtRun(
      * read from the default location `.ort.env.yml`.
      */
     val environmentConfigPath: String? = null,
+
+    /**
+     * The optional trace ID that is assigned to this run. This is generated when the run is created. It can be used to
+     * correlate the logs from different components that are taking part in processing of the run.
+     */
+    val repositoryIds: List<Long>? = null,
```

Comment:
As a list can always be empty, making it nullable additionally should have a clear semantic purpose. I guess we could just make this non-nullable and empty by default, and by convention treat empty `repositoryIds` as *all* `repositoryIds`.